src/elf.rs

```diff
@@ -498,6 +498,31 @@ impl<E: UserDefinedError, I: InstructionMeter> Executable<E, I> {
         })
     }
 
+    /// Calculate the total memory size of the executable
+    pub fn mem_size(&self) -> u64 {
+        let total = std::mem::size_of_val(&self)
```

Comment:
Inconsistent style: Use `std::mem::size_of::<Self>()` here as well?

Reply:
Like I said, quick and dirty to get a rough estimate